cmd/daemon/main.go

```diff
@@ -417,6 +422,66 @@ func loadConfig(cfg *Config) error {
 	return nil
 }
 
+func (cfg *Config) ConfigPath() string {
+	return filepath.Join(cfg.ConfigDir, "config.yml")
+}
+
+func (cfg *Config) StatePath() string {
+	return filepath.Join(cfg.ConfigDir, "state.json")
+}
+
+func (cfg *Config) CredentialsPath() string {
+	return filepath.Join(cfg.ConfigDir, "credentials.json")
+}
+
+type AppState struct {
+	DeviceId    string                `json:"device_id"`
+	Credentials storedCredentialsFile `json:"credentials"`
```

Comment:
Can you rename `storedCredentialsFile` to something that makes more sense now or just inline it if it's not used anywhere else?